runners/flink/src/main/java/org/apache/beam/runners/flink/translation/functions/FlinkExecutableStageFunction.java

```diff
@@ -331,10 +331,8 @@ private void fireTimer(
   public void close() throws Exception {
     // close may be called multiple times when an exception is thrown
     if (stageContext != null) {
-      try (@SuppressWarnings("unused")
```

Comment:
Are these no longer needed by any of our tools?

Reply:
This would be a javac or findbugs issue, right?

Reply:
In this particular place, these @SuppressWarnings were removed because no longer 
needed (that is, without them, none of our tool gives complains).  
SuppressWarnings is something that less is always better, thus I took the liberty removing them. :-)

One side note, and also to @kennknowles 's question: Yes, it should be either by javac or findbugs. checkstyle does not cover unused.  The fact is , check/warning on unused-local-variable is completely turned off everywhere in our code base. 
I tested and verified by intentionally adding ones but no tool complains on that, thus I assume it is intended behavior. If it is not expected (is that so?), that deserves further investigation.

Reply:
Sounds good to me. If we care about unused variables, we can turn it on again. Removed the not-needed annotation is an improvement.